addons/website/static/src/scss/website.scss

```diff
@@ -100,18 +100,22 @@ $-seen-urls: ();
     @include print-variable('custom-colors', $custom-colors);
```

Comment:
Isn't that change too much for a stable version ?
I just did the following:
- In a V15 (without your changes)
- Apply your steps to reproduce
=> I have a white BG, I can't see the BG image
- Simulate the weekly update and so cherry pick your commit
=> I now have my BG image as a background instead of white. 

I'm afraid that if we merge that we will have some clients complaining about. "I did nothing and my website design is broken".  Just because they played with this option one day.

Maybe what we should do instead is apply a specific class when the BG background is apply and only do your change if this class is there. What do you think ?

Reply:
I don't think such a case is likely to exist since it just creates flickering when you have this configuration without the fix, see: https://app.screencastify.com/v3/watch/8apPUKgeWYMvPDReLE8X

It's why I did not care about the sudden change.

But it's a good remark.
I pushed [this diff](https://github.com/odoo/odoo/compare/86601746f1acb88df7ada80bbb1aa25b40356d99..05691a526991a386ade88a963ec485f98d6578ac), which is using a new scss key/value (and not a class which would not be ideal to add stuff in the DOM, probably annoying in upgrade too need to get rid of it) in case you really want it.
I'll add comment and adapt commit msg regarding your answer if we keep the diff :+1:

Reply:
I suppose you are right. Nobody used the options and leave the website with a flickering on load. Lets remove the diff you just added and then I will merge :+1: Thanks

Reply:
Removed